WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -408,6 +412,13 @@ class GutenbergViewController: UIViewController, PostEditor, FeaturedImageDelega
         }
     }
 
+    override func present(_ viewControllerToPresent: UIViewController, animated flag: Bool, completion: (() -> Void)? = nil) {
+        super.present(viewControllerToPresent, animated: flag, completion: completion)
+
+        let presentedView = presentedViewController?.view
+        presentedView?.tintColor = .editorPrimary
```

Comment:
```suggestion
        // Update the tint color for React Native modals when presented
        let presentedView = presentedViewController?.view
        presentedView?.tintColor = .editorPrimary
```

It may be worth leaving an inline comment explaining what this accomplishes for posterity.

Reply:
Committed, thank you!